locust/runners.py

```diff
@@ -49,6 +51,27 @@ def on_hatch_complete(user_count):
                 self.stats.reset_all()
         events.hatch_complete += on_hatch_complete
 
+    @staticmethod
+    def build_locust_classes_by_name(locust_classes):
+        """
+        Returns a dictionary of the form {locust_class_name: locust_class}
+        """
+        if isinstance(locust_classes, type):
+            return {locust_classes.__name__: locust_classes}
+        elif isinstance(locust_classes, collections.Iterable):
+            return {locust.__name__: locust for locust in locust_classes}
+        else:
+            return {}
```

Comment:
won't this case become an error at the UI level because there will be no locusts to select? I'm trying to imagine a case where it hits here, though, because iirc there's validation prior to this point.

Reply:
That's a fair point, I can raise an exception here, and catch the exception in the runner tests.